plugins/node/opentelemetry-instrumentation-koa/README.md

```diff
@@ -38,7 +38,7 @@ registerInstrumentations({
 });
 ```
 
-See [`examples/koa`](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/examples/koa) for a short example using both Koa and @koa/router
+See [`plugins/node/opentelemetry-instrumentation-koa/examples`](https://github.com/open-telemetry/opentelemetry-js-contrib/tree/main/plugins/node/opentelemetry-instrumentation-koa/examples) for a short example using both Koa and @koa/router
```

Comment:
Same a previous comment. I think the `plugins/node/opentelemetry-instrumentation-koa/` prefix is a bit too verbose and noisy for the README
Just my personal preference :)